.github/workflows/tests.yml

```diff
@@ -28,12 +29,59 @@ jobs:
       run: |
         cd docs
         make html
+
+  ui-test:
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 40
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-22.04]
+        python: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
```suggestion
        python: ["3.8", "3.9", "3.10", "3.11"]
```

Reply:
I would even recommend the same pattern as in the `python` job where all Python versions are run on Linux but only oldest and latest is run on Windows.